go.sum

```diff
@@ -0,0 +1,18 @@
+github.com/davecgh/go-spew v1.1.0 h1:ZDRjVQ15GmhC3fiQ8ni8+OwkZQO4DARzQgrnXU1Liz8=
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
How did we end up with go-spew here? It's very useful for debugging, but I don't think we have an explicit dependency in the code...

Reply:
The go.sum file came from running `go mod tidy` we can manually clean this up but it will keep getting added back when issuing commands like `go build` or `go get` since it will see the package and include it.

https://github.com/golang/go/issues/26913#issuecomment-411976222 covers the current state of build vs test dependencies and the hopeful fixes that it should get in the future.

Reply:
Oh, it comes from tests. That makes sense. It doesn't really matter - if I understand correctly `go.sum` is just used for verification that the dependencies are legitimate.